magellan-library/src/main/java/com/wealthfront/magellan/core/Journey.kt

```diff
@@ -3,13 +3,13 @@ package com.wealthfront.magellan.core
 import android.view.LayoutInflater
 import androidx.viewbinding.ViewBinding
 import com.wealthfront.magellan.ScreenContainer
-import com.wealthfront.magellan.lifecycle.lifecycle
+import com.wealthfront.magellan.lifecycle.attachFieldToLifecycle
 import com.wealthfront.magellan.navigation.LinearNavigator
 
 public abstract class Journey<V : ViewBinding>(
-  createBinding: (LayoutInflater) -> V,
-  container: V.() -> ScreenContainer
-) : Step<V>(createBinding) {
+  createBindingProvider: (LayoutInflater) -> V,
+  containerProvider: V.() -> ScreenContainer
+) : Step<V>(createBindingProvider) {
```

Comment:
What was the rationale for changing `createBinding` -> `createBindingProvider`?

My suggestion would be `inflateBinding` and `getContainer`, IMO they read better that way (`inflateBinding = MyViewBinding::inflate` in subclasses and `val binding = inflateBinding(layoutInflater)` in this class)